cmd/zpool/zpool_main.c

```diff
@@ -7830,6 +7818,16 @@ find_command_idx(char *command, int *idx)
 	return (1);
 }
 
+static void
+pre_import_callback(nvlist_t *config)
```

Comment:
The issue here is that when the mmp check is required we delay in the tryimport ioctl for a long time.  (It's necessary to do the check in tryimport so `zpool import` can accurate report the pool status, fixed in PR to be updated shortly).  The idea was to provide the user a message before entering the system call to explain the delay.  However, in practice I'm not sure it's really that useful and since it can never be 100% accurate I'm leaning towards dropping it too.